go.mod

```diff
@@ -3,6 +3,8 @@ module github.com/ClusterLabs/ha_cluster_exporter
 go 1.12
 
 require (
+	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
+	github.com/modern-go/reflect2 v1.0.1 // indirect
```

Comment:
could you run `go mod tidy` to get rid of unnecessary indirects?

Reply:
yop